catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -95,6 +106,10 @@ public void setSrc(String src) {
     this.src = src;
   }
 
+  public void setSrcs(List<String> srcs) {
```

Comment:
✏️ It would be less confusing now if we merged `src` and `srcs`.

Reply:
This was what I originally did. @andrewkfiedler suggested supporting both so we don't have unwanted side effects related to the existing front end code that uses `src`

Reply:
I think this will increase the tech debt of this class but since you have already changed it once we can leave it as it is now.